src/components/InfoAboutPerson.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function InfoAboutPerson({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {person.name ? `My name is ${person.name}` : null}
+      </h2>
```

Comment:
`name` is required. It cannot be null
```suggestion
      <h2 className="Person__name">
        My name is {person.name}
      </h2>
```